com.synology.CloudStationBackup.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.synology.CloudStationBackup
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: synology-cloud-station-backup
+tags: [proprietary]
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: synology-cloud-station-backup
+    buildsystem: simple
+    build-commands:
+      - install -D synology-cloud-station-backup.sh /app/bin/synology-cloud-station-backup
+      - install -Dm644 com.synology.CloudStationBackup.desktop -t /app/share/applications
+      - install -Dm644 com.synology.CloudStationBackup.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 sico_72_cloud_station.png /app/share/icons/hicolor/64x64/apps/com.synology.CloudStationBackup.png
+      - install -Dm644 sico_128_cloud_station.png /app/share/icons/hicolor/128x128/apps/com.synology.CloudStationBackup.png
+      - install -Dm644 sico_256_cloud_station.png /app/share/icons/hicolor/256x256/apps/com.synology.CloudStationBackup.png
+      - install -D apply_extra -t /app/bin
+      - cp /usr/bin/ar /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so
```

Comment:
Nitpick: You can use `$FLATPAK_ARCH` instead.

Reply:
Remove the line
`       - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so ` 
as it seems to not be needed